testing/integration/fqdn_test.go

```diff
@@ -32,6 +32,7 @@ import (
 
 func TestFQDN(t *testing.T) {
 	info := define.Require(t, define.Requirements{
+		Group: "fqdn",
```

Comment:
Is a test by itself in its own group equal to setting `Isolate: true`? Do we need both ways to control this?

Reply:
Actually I don't think that we need both. I remove isolate as you can now just place an individual test in its own group.